src/App.js

```diff
@@ -22,7 +22,6 @@ state = {
 	 <input type="text" name="age" placeholder="age" onChange={this.inputChange} value={this.state.age}/>
```

Comment:
Ok. i have a question here, instead of having to call onChange can we do value change, i know that will be tricky but i think its better design, since we are mostly dealing with text and content changes